tools/src/commands/ValidateWorkspaceDependencies.ts

```diff
@@ -0,0 +1,72 @@
+import { Command } from '@expo/commander';
+import path from 'path';
+
+import { getExpoRepositoryRootDir } from '../Directories';
+import { getInfoAsync } from '../Workspace';
+
+export default (program: Command) => {
+  program
+    .command('validate-workspace-dependencies')
+    .alias('vwd')
+    .description('Verifies if all workspace packages are linked from monorepo.')
+    .asyncAction(actionAsync);
+};
+
+async function actionAsync() {
+  const workspaces = await getWorkspacesAsync();
```

Comment:
nit but probably a good idea, rename variables too:
```suggestion
  const workspacePackages = await getWorkspacePackagesAsync();
```
etc...